configure.ac

```diff
@@ -1776,9 +1776,8 @@ fi
 
 # SHAKE256
 AC_ARG_ENABLE([shake256],
-    [AS_HELP_STRING([--enable-shake256],[Enable wolfSSL SHAKE256 support (default: enabled on x86_64/aarch64)])],
-    [ ENABLED_SHAKE256=$enableval ],
-    [ ENABLED_SHAKE256=$ENABLED_SHA3 ]
```

Comment:
SHAE256 is inly available if SHA-3 is compiled in.
Make sure SHA-3 is compiled in when SHAKE256 is required.

Reply:
Maybe put a warning in somewhere if SHAKE256 is enabled by SHA-3 not.

Reply:
Is the existing check somehow insufficient?

https://github.com/wolfSSL/wolfssl/pull/4080/files#diff-49473dca262eeab3b4a43002adb08b4db31020d190caaad1594b47f1d5daa810R3211-R3214

Reply:
Check in there on line 3211 of configure.ac.